### PR TITLE
Rename to kinto-http.js

### PR DESCRIPTION
**Pull request by Natim**
r? @n1k0

### CODE REVIEW

**Comment by n1k0** [comment]
On top of my head:

- should we update the main constructor name? it's still `KintoClient` atm  
- we should pay attention at how this change impacts Firefox (there's a target build for it)
- we should deprecate `kinto-client` on npm and publish `kinto-http`

And I may be missing other important stuff to do before processing changing the name.

File: README.md [hunk 1]
```diff
@@ -60,30 +60,30 @@ Read the [API documentation](https://doc.esdoc.org/github.com/Kinto/kinto-client
...
 ```
-$ npm install kinto-client --save
+$ npm install kinto-http --save
 ```
 
 Then (ES6):
 
 ```js
-import KintoClient from "kinto-client";
+import KintoClient from "kinto-http.js";
```
Comment (n1k0) [inline README.md]: 3 lines above you installed it using "$ npm install kinto-http --save" so it's `import KintoClient from "kinto-http"`

File: README.md [hunk 2]
```diff
@@ -60,30 +60,30 @@ Read the [API documentation](https://doc.esdoc.org/github.com/Kinto/kinto-client
...
 ```
 
 Or (ES5):
 
 ```js
-var KintoClient = require("kinto-client").default;
+var KintoClient = require("kinto-http.js").default;
```
Comment (n1k0) [inline README.md]: Ditto

**Comment by Natim** [comment]
For the constructor I think we can keep `KintoClient` I wish we do not break to much existing applications.

**Comment by Natim** [comment]
> we should deprecate kinto-client on npm and publish kinto-http

As soon as we merge this we will.

**Comment by n1k0** [comment]
I'm about to r+ but would really appreciate @leplatrem to confirm the patch is okay wrt Gecko integration.

**Comment by Natim** [comment]
Sure :0

**Comment by leplatrem** [comment]
R+ Gecko does not rely on NPM but on the exported symbol.
Please update the wiki page mozilla.org Firefox/Kinto

**Comment by Natim** [comment]
@n1k0 I would be happy to pair with you on that release to learn about how to deprecate a module with npm.

**Comment by n1k0** [comment]
Here's a good guide for deprecation https://www.gregjs.com/javascript/2016/how-to-rename-a-published-npm-module/

**Comment by n1k0** [comment]
Merging, deprecation of `kinto-client` and publication of `kinto-http` to follow.

**Comment by DirtyF** [comment]
FYI `kinto-client.js` is still mentioned on http://www.kinto-storage.org/

**Pull request merged**